src/gt4py/next/program_processors/runners/dace_fieldview/gtir_builtin_translators.py

```diff
@@ -161,8 +161,8 @@ def translate_as_field_op(
     node: gtir.Node,
     sdfg: dace.SDFG,
     state: dace.SDFGState,
-    sdfg_builder: gtir_to_sdfg.SDFGBuilder,
-    reduce_identity: Optional[gtir_to_tasklet.SymbolExpr],
+    sdfg_builder: gtir_sdfg.SDFGBuilder,
+    reduce_identity: Optional[gtir_dataflow.SymbolExpr],
 ) -> FieldopResult:
     """Generates the dataflow subgraph for the `as_fieldop` builtin function."""
```

Comment:
You describethe format of the argument below in a comment.
I would put them into the doc string, because this is the central place you start looking for.